.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,76 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+on:
+  pull_request:
+  push:
+    branches:
+      - next
+    tags:
+      - "**"
+
+name: "Continuous Integration"
+
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v1.1.0
+
+      - name: "Install PHP with extensions"
+        uses: shivammathur/setup-php@1.5.1
+        with:
+          php-version: 7.1
```

Comment:
Are we running phpcs only for 7.1 at this point? No idea if we could actually have a diff there based on a PHPversion. Dont really think so though

Reply:
@pimjansen 

I think it suffices to run a coding standards fixer on the lowest supported version of PHP.